docs/website/layouts/partials/meta.html

```diff
@@ -52,6 +52,11 @@
 
 <!-- Algolia metadata -->
 <meta name="docsearch:language" content="en" />
+{{- if $isLatest -}}
+<link rel="canonical" href="{{ $latest }}">
+{{- else -}}
+<link rel="canonical" href="{{ $version }}">
+{{- end -}}
 <meta name="docsearch:version" content="{{ $version }}" />
```

Comment:
That line is ok, though?

Reply:
Oops, more haste, less speed. I've fixed this in c2b00a1ee650784e48b4e5862638fefc6c7ccc08